tests/suites/config.js

```diff
@@ -0,0 +1,36 @@
+module.exports = [{
+	deviceType: `raspberrypi3`,
+	suite: `${__dirname}/../suites/os`,
+	config: {
+		networkWired: false,
+		networkWireless: true,
+		downloadType: 'local',
+		interactiveTests: false, // redundant
+        balenaApiKey: process.env.API_KEY,
+        balenaApiUrl: 'balena-cloud.com',
+        organization: process.env.ORG
+	},
+	image: `${__dirname}/balena.img.gz`,
+	workers: {
+		balenaApplication: 'testbot-rig',
+		apiKey: process.env.BALENA_CLOUD_API_KEY,
+	},
+},
+{
+	deviceType: `raspberrypi3`,
+	suite: `${__dirname}/../suites/hup`,
+	config: {
+		networkWired: false,
+		networkWireless: true,
+		downloadType: 'local',
+		interactiveTests: false, // redundant
+        balenaApiKey: process.env.API_KEY,
+        balenaApiUrl: 'balena-cloud.com',
+        organization: process.env.ORG
+	},
+	image: `${__dirname}/balena.img.gz`,
+	workers: {
+		balenaApplication: 'testbot-rig',
+		apiKey: process.env.BALENA_CLOUD_API_KEY,
+	},
+}];
```

Comment:
@robertgzr after a discussion with @rcooke-warwick this morning we decided to pre-define the Leviathan config.js for now rather than generate it dynamically in Jenkins. This is cleaner as some suites, like hup, will require custom configuration.

For example, I believe the hup suites requires different values for `downloadType` and `image` but I wasn't 100% sure so I left the defaults and you can update accordingly and squash the commit.

Reply:
thanks, added a fixup commit for that, I'll squash them later.

Reply:
we will also have to think of a way to get the `balena-image.docker` artifact gzip'ed for the test suite to pick up...

Reply:
Is it the FROM image or the TO image that needs to be gzipped? I believe it's the TO image and you're importing it into a local docker instance? Why not add another image unpack option to leviathan that is copy only?

Reply:
@klutchell it's the TO image. And yeah I was wondering why we didn't just add a `copy` unpack option... I guess it saves us some bandwidth and cuts down on time spent uploading...

Reply:
@rcooke-warwick thoughts on adding `downloadType=copy` for cases where we are using the `balena-image.docker` artifact and not the `balena-img.gz` artifact?

Reply:
This is so we don't have to turn `balena-image.docker` into `balena-image.docker.gz` @klutchell?

Reply:
> This is so we don't have to turn `balena-image.docker` into `balena-image.docker.gz` @klutchell?

Pretty much, we don't want to change/break the existing artifacts if possible. We could maybe add a step to the Leviathan Jenkins job to gzip the file first if that's better?

Reply:
@klutchell Just double checking something - I have a feeling that the client that sends the tests+image to the testbot already has some utility to gzip the file before sending it, I'll have a quick look and report back

Reply:
@klutchell so we can give a non `.gz` file, and the client will `gz` it anyway before sending it to the testbot - is this ok? To get that to actually work will require a 1 line fix in leviathan, will make the PR

Reply:
wow I didn't know that, great!

Reply:
@rcooke-warwick @klutchell with an non `.gz` file I get:
```
[8f854c3-hup] Unpacking the operating system
[8f854c3-hup] # Subtest: Hostapp update suite
[8f854c3-hup]     not ok 1 - incorrect header check
[8f854c3-hup]       ---
[8f854c3-hup]       at:
[8f854c3-hup]         line: 164
[8f854c3-hup]         column: 17
[8f854c3-hup]         file: zlib.js
        function: Zlib.zlibOnError
[8f854c3-hup]         method: onerror
      cause:
[8f854c3-hup]         !error
[8f854c3-hup]         name: Error
[8f854c3-hup]         message: incorrect header check
[8f854c3-hup]         stack: |-
[8f854c3-hup]           Error: incorrect header check
[8f854c3-hup]               at Zlib.zlibOnError [as onerror] (zlib.js:164:17)
[8f854c3-hup]         errno: -3
[8f854c3-hup]         code: Z_DATA_ERROR
[8f854c3-hup]       isOperational: true
[8f854c3-hup]       errno: -3
[8f854c3-hup]       code: Z_DATA_ERROR
[8f854c3-hup]       test: Hostapp update suite
[8f854c3-hup]       ...
[8f854c3-hup] 
[8f854c3-hup]     1..1
[8f854c3-hup]     # failed 1 test
[8f854c3-hup] not ok 1 - Hostapp update suite # time=7013.281ms
[8f854c3-hup] 
[8f854c3-hup] not ok 2 - cannot create subtest after parent test end
[8f854c3-hup]   ---
  stack: |
[8f854c3-hup]     treeExpander (lib/common/suite.js:128:19)
[8f854c3-hup]     Test.testNode.test (lib/common/suite.js:159:13)
  test: Hostapp update suite
  ```

Reply:
looks like the `gzunip` unpack type at least doesn't do that

Reply:
@robertgzr it will require a 1 line fix in leviathan to work! I'm not at my latpop now but I'll put the PR in for it tomorrow morning promise!

Reply:
![image](https://user-images.githubusercontent.com/20458272/118274154-c6e42f00-b492-11eb-9842-d14660b5b85b.png)

Reply:
the gzipping is fixed in https://github.com/balena-os/leviathan/pull/393, which is about to be merged + deployed into the balena app by @vipulgupta2048.